CAIPs/caip-380.md

```diff
@@ -0,0 +1,178 @@
+---
+caip: CAIP-380
+title: Portable Proof
+author: Chris Leal (@cl34l)
+discussion-to: https://github.com/ChainAgnostic/CAIPs/pull/380
+status: Draft
+type: Standard
+created: 2025-10-01
+updated: 2025-10-01
+requires: [2,10]
+---
+
+## Simple Summary
+
+A canonical, chain-agnostic, wallet-signed envelope that applications validate once off-chain and can reference anywhere via a deterministic `qHash` anchor.
+Identities follow CAIP-10 (`did:pkh`) and chains follow CAIP-2.
+
+## Abstract
+
+Defines a chain-agnostic, wallet-signed proof object anchored by a SHAKE-256 digest (`qHash`).
+Applications validate once off-chain; the same `qHash` may be surfaced on-chain for indexing/transport via [EIP-7683]–compatible vouchers.
+
+## Motivation
+
+Establish a canonical, deterministic envelope that can be validated once and referenced anywhere (off-chain or on-chain) without binding to any vendor, domain, or transport.
+Align identities with CAIP-10 and chain context with CAIP-2 to avoid namespace collisions.
+
+## Rationale
+
+- Use [EIP-191] for broad wallet compatibility; support [EIP-1271] and detect [EIP-6492] to cover smart accounts and pre-deploy signatures.
+- Keep `qHash` anchored to a canonical subset to ensure stable, cross-environment equivalence and idempotency.
+- Treat vouchers as optional [EIP-7683]–compatible artifacts keyed by `qHash` to enable cross-chain transport without constraining settlement designs.
+
+## Specification
+
+- **Terminology:**
+  - Envelope: the signed data structure defined in this document
+  - Anchor: `qHash` computed over the canonical subset
+  - Verifier: logic module identified by `verifierId`
+  - Voucher: optional on-chain artifact ([EIP-7683]–compatible) keyed by `qHash`
+
+- **Identifiers:**
+  - Chains: `eip155:<chainId>` per CAIP-2 (numeric `chainId` on-wire)
+  - Accounts: `did:pkh:eip155:<chainId>:<address>` per CAIP-10
+- **Envelope (top-level):**
+  - Note that the fields `signedMessage` and `signatureMethod` are OPTIONAL (see Conformance).
+
+```json
+  {
+    "did": "did:pkh:eip155:1:0xabc000000000000000000000000000000000def0",
+    "qHash": "0x<shake256-hex>",
+    "verifierIds": ["ownership-basic"],
+    "data": {},
+    "signature": "0x…",
+    "signedMessage": "Portable Proof Verification Request\nWallet: 0xabc000000000000000000000000000000000def0\nChain: 1\nVerifiers: ownership-basic\nData: {}\nTimestamp: 1730000000000",
+    "signedTimestamp": 1730000000000,
+    "chainId": 1,
+    "signatureMethod": "eip191",
+    "options": {},
+    "meta": {}
+  }
+```
+
+- **Canonical signing (MUST):** [EIP-191] (also known as `personal_sign`). Exact six-line signer string structure (line 1 is a fixed context label; default label shown):
+  - Freshness: reject if older than 5 minutes or >1 minute in the future
+  - Smart-accounts: support [EIP-1271];
+  - detect [EIP-6492] wrappers.
+
+```sh
```

Comment:
my markdown linter was requiring a language here, feel free to put a better one than shell formatting if you can think of one